vllm_ascend/quantization/quant_config.py

```diff
@@ -36,6 +34,7 @@
 from vllm.model_executor.parameter import PerTensorScaleParameter
 from vllm.model_executor.utils import set_weight_attrs
 
+from ..ops.fused_moe import AscendUnquantizedFusedMoEMethod
```

Comment:
I prefer to use absolute path references here
```suggestion
from vllm_ascend.ops.fused_moe import AscendUnquantizedFusedMoEMethod
```

Reply:
ok. i will modify it